src/arrays/qucoeffs.jl

```diff
@@ -0,0 +1,69 @@
+import Base: transpose,
+    ctranspose,
+    conj,
+    size,
+    ndims,
+    length,
+    getindex,
+    setindex!
+
+############
+# QuCoeffs #                 
+############
+    abstract ConjBool{Conj}
+    abstract TranBool{Tran}
```

Comment:
Thanks for the clarification. I should probably have a look into JuliaLang/julia#6837 again to be on the same page. 

>a transpose would cause a copy, a conjugation would cause a copy, but the ctranspose would cause a view. Thus, doing conj(transpose(arr)) would be a double copy operation, while ctranspose(arr) would be a no-copy operation, which seems ugly/confusing to me.

Maybe this would indeed be too confusing. I was just thinking that we mainly want `v'=dual(v)` which typically just happens to be `ctranspose(v)=conj(transpose(v))` (i.e. all three should return a copy, but `dual` would return a view).